packages/analytics-js-common/src/utilities/eventMethodOverloads.ts

```diff
@@ -55,80 +55,87 @@ const pageArgumentsToCallOptions = (
   options?: Nullable<ApiOptions> | ApiCallback,
   callback?: ApiCallback,
 ): PageCallOptions => {
-  const payload: PageCallOptions = {};
+  const payload: PageCallOptions = {
+    category: category as string,
+    name: name as string,
+    properties: properties as Nullable<ApiObject>,
+    options: options as Nullable<ApiOptions>,
+  };
 
   if (isFunction(callback)) {
-    payload.category = category as string;
-    payload.name = name as string;
-    payload.properties = clone(properties as Nullable<ApiObject>);
```

Comment:
why we do not clone anymore? we need this to happen as soon as we start accessing consumer provided values

Reply:
what is the bug that we are trying to fix and we had to refactor this critical parth of the app?